### PR TITLE
[PoC][Feature] Add new action for room scan and waiting page

### PR DESCRIPTION
**Pull request by shpran**
# PoC for [TR-5940](https://oat-sa.atlassian.net/browse/TR-5940)

# Changes
* Added new action `startRoomScan`
* Added waiting page

## Companion PRs
* https://github.com/oat-sa/generis/pull/1107
* https://github.com/oat-sa/extension-remote-proctoring/pull/89

[TR-5940]: https://oat-sa.atlassian.net/browse/TR-5940?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jsconan — APPROVED** [review]
Nice!

The UI will need to be refined for using the TAO style, but this looks already great!

File: views/templates/DeliveryServer/waiting_page.tpl [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            box-shadow: rgba(60, 64, 67, .3) 0 4px 4px 0, rgba(60, 64, 67, .15) 0 8px 12px 6px;
+        }
+
+        .button:disabled {
+            box-shadow: rgba(60, 64, 67, .3) 0 1px 3px 0, rgba(60, 64, 67, .15) 0 4px 8px 3px;
+        }
+    </style>
+</head>
+<body>
+<a class="button" href="<?= get_data('delivery-execution-url') ?>"><?=__('Start')?></a>
```
Comment (gabrielfs7) [inline views/templates/DeliveryServer/waiting_page.tpl:86]: - [ ] Lets please align the button in the middle.

Nice to have: 

- Add the topbar without the other buttons, keep only the logo.
- Add same button style that we have on select tests page.

Comment (jsconan) [reply views/templates/DeliveryServer/waiting_page.tpl]: All done!

**Review by gabrielfs7 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**